docker/dockerfiles/toolchains/toolchains-centos7.Dockerfile

```diff
@@ -81,3 +81,4 @@ RUN ARCH=`uname -m` ; if [[ $ARCH == "aarch64" ]] ; then DOWNLOAD_URL=http://cdn
 ENV STARROCKS_GCC_HOME=${GCC_INSTALL_HOME}
 ENV JAVA_HOME=/usr/lib/jvm/java-11
 ENV MAVEN_HOME=${MAVEN_INSTALL_HOME}
+ENV LANG=en_US.utf8
```

Comment:
do we have this locale generated for centos7?

Reply:
Sorry for the late reply, I have checked the toolchains-centos7 image and it has en_US.utf8.

![image](https://github.com/user-attachments/assets/fa7e9f93-b6f3-4cda-9f46-3213c9383d54)